plugins/woocommerce-admin/client/header/index.js

```diff
@@ -93,12 +95,18 @@ export const Header = ( { sections, isEmbedded = false, query } ) => {
 		}
 	}, [ isEmbedded, sections, siteTitle ] );
 
-	const tasksReminderFeature =
-		window.wcAdminFeatures[ 'tasklist-setup-experiment-1' ];
+	const { hasTasksReminderFeature } = useSelect( ( select ) => {
+		const taskLists = select( ONBOARDING_STORE_NAME ).getTaskLists();
+		return {
+			hasTasksReminderFeature: taskLists.some(
+				( list ) => list.id === 'setup_experiment_1'
+			),
+		};
+	} );
```

Comment:
Curious if we want to make use of the `visibleTaskListIds` here instead, so there is not delay -> `getAdminSetting( 'visibleTaskListIds', [] )`.
But it's optional given this component will require the Task list data anyway, so it likely won't improve the loading time that much.

Reply:
Good thought.  I think that's a good idea, but inclined to leave for a follow-up in the interest of time with the code freeze upon us.